makefile

```diff
@@ -4,7 +4,11 @@
 install:
 	npm prune && npm install
 	cd node_modules/.bin; ./jspm install && ./jspm dl-loader && ./jspm clean
-	grunt copy
+	grunt uglify:conf
```

Comment:
yeah, that `copy` line was wrong, should always have been `uglify:conf` (whether _that's_ correct is another matter...)